crates/bevy_render/src/texture/image.rs

```diff
@@ -831,6 +831,8 @@ impl RenderAsset for Image {
         let texture = render_device.create_texture_with_data(
             render_queue,
             &self.texture_descriptor,
+            // TODO: Is this correct? Do we need to use `MipMajor` if it's a ktx2 file?
+            wgpu::util::TextureDataOrder::LayerMajor,
```

Comment:
This is a new argument, I provided it with the default value so it should work the same way it did before. :)

We can explore forwarding this configuration to user API in further PRs.